### DISCUSSION TITLE
[suggestion] Simplify # region line

### DISCUSSION THREAD

**Discussion by surfsky** [dotnet/csharplang#3498] [Language Ideas]
Maybe we can simply ```#region``` expression to:
- Delete the endregion line
- You can free drag and sort code region in vs(code)
eg.
``` csharp
# region 1
....
# region 2
....
# region 3
```

or rename to part

``` csharp
# part 1
...
# part 2
...
# part 3
```

**Comment by yaakov-h** [comment]
Dragging regions could probably be an IDE enhancement without requiring a language change.

As for dropping `#endregion`, how would you now know where a region ends?

**Comment by surfsky** [comment]
The region end is just the new #region line or end of file
Or we can rename it to '''# part''' to avoid confliction.

**Comment by miloush** [comment]
That would break existing nested regions.
